src/car/car.service.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @typedef {import('../../types/src/car/car.service').Deps} Deps
+ * @typedef {import('../../types/src/car/car.service').GetCarsListParams} GetCarsListParams
+ * @typedef {import('../../types/src/car/car.service').CarService} CarService
+ * @typedef {import('../../types/src/car/car.repository').Car} Car
+ */
+
+import { partial } from '@oldbros/shiftjs';
+
+/**
+ *
+ * @param {Deps} deps
+ * @param {GetCarsListParams} getListParams
+ * @returns {Promise<Car[]>}
```

Comment:
return type not contains {
    cars,
    currentPage: page,
    totalPages: Math.ceil(carsCount / perPage) || 1,
  }